bitnami/kafka/values.yaml

```diff
@@ -506,6 +506,10 @@ nodeSelector: {}
 ##
 tolerations: []
 
+## Seconds the pod needs to gracefully terminate
+# terminationGracePeriodSeconds: 30
```

Comment:
Could you add the link to the documentation about `terminationGracePeriodSeconds`?
``` yaml
  ## Configure the grace time period for sig term
  ## ref: https://kubernetes.io/docs/concepts/containers/container-lifecycle-hooks/#hook-handler-execution
  ##
```

Reply:
Have added the comments according to your suggestions. Thank you